api/errors.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
That's a good idea fam.

On Wed, Apr 13, 2016, 10:30 PM Dan Cadden <notifications@github.com> wrote:

> In api/errors.go
> <https://github.com/skeswa/gophr/pull/54#discussion_r59656452>:
>
> > @@ -0,0 +1,39 @@
> > +package main
>
> Should this not be a subsection of common? `common/errors/api-errors.go?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/skeswa/gophr/pull/54/files/348e491f3761f5491d6bf61bee64f0db535693b6#r59656452>
>